lib/validations.js

```diff
@@ -760,10 +760,21 @@ function ValidationError(obj) {
 util.inherits(ValidationError, Error);
 
 var errorHasStackProperty = !!(new Error).stack;
+var inspect = require('util').inspect;
+
+function formatProperty(obj, propertyName) {
+  var value = obj[propertyName];
+  if ((typeof value) === 'function')
+    return propertyName
+  else if ((typeof value) === 'string')
+    return propertyName + ': ' + inspect(value.substring(0, 32));
+  else
+    return propertyName + ': ' + inspect(value);
+}
 
```

Comment:
What do you think of this?  I noticed we were producing some strange output in the case of nested documents (which have a type of `function`).  I don't know how to output a sensible output in that case... but I think it's an edge case that can be left for another day.

For strings, objects, and undefined/null we now have some pretty clear output.